packages/firebase_vertexai/firebase_vertexai/example/lib/main.dart

```diff
@@ -137,7 +140,8 @@ class _ChatWidgetState extends State<ChatWidget> {
   );
 
   Future<void> initFirebase() async {
-    await Firebase.initializeApp();
+    // ignore: avoid_redundant_argument_values
+    await Firebase.initializeApp(options: options);
```

Comment:
does firebase need to be initialised if it doesn’t need firebase app?

Reply:
```
  static FirebaseVertexAI get instance {
    return FirebaseVertexAI.instanceFor(
      app: Firebase.app(),
    );
  }
```

The FirebaseVertexAi instance is using the defaut Firebase.app so it needs to be instantiated.

Reply:
I would've thought an exception would throw without any firebase options running on web?

Reply:
I'm not sure about that, the SDK might try to initialize from meta data in the header if no options are provided in the Dart side, then fail if they cannot find it in HTML header.